src/users/users.model.ts

```diff
@@ -31,4 +31,12 @@ export class Users extends Model {
 
   @BelongsToMany(() => Groups, () => UsersGroups)
   groups: Groups[];
+
+  static async isLoginUniq(usersRepository: Repository<Users>, login: string): Promise<boolean> {
+    return !usersRepository.findOne({
+      where: {
+        login,
+      },
+    });
+  }
```

Comment:
not sure if it is a model level helper...

Reply:
fixed